src/WellIO.jl

```diff
@@ -1,5 +1,47 @@
 module WellIO
 
-# Write your package code here.
+using FileIO
+using Tables
 
+"""
+    load(file)
+Load log well file from `filepath` and load it in defined structure.
+"""
+function load(filepath::String)
+    file = FileIO.query(filepath)
+    data = open(file) do f
+            read(f, String)
+        end
+    #Splitting the data into table and rest of information
+    datatable_block = match(r"~A(.*?)\Z"s, data)
+    info_block = data[1:datatable_block.offset]
+    #Extracting specific information from info_block
+    version_block = match(r"~V(.*?)~"s, info_block)
+    wellinfo_block = match(r"~W(.*?)~"s, info_block)
+    curveinfo_block = match(r"~C(.*?)~"s, info_block)
+    paraminfo_block = match(r"~P(.*?)~"s, info_block)
+    otherinfo_block = match(r"~O(.*?)~"s, info_block)
```

Comment:
It seems that all these match calls could be a single match call with multiple `(.*?)` groups? I would try to write it as:

```julia
m = match(r"~A(.*?)~V(.*?)~...", data)
```

if there is guarantee that these sections always appear in these order. Can you please rename the variables names to something more sensible? Avoid redundant naming like the `_block` suffix. Variables names should be informative and as short as possible.